src/client/Microsoft.Identity.Client/TokenCacheNotificationArgs.cs

```diff
@@ -146,6 +147,44 @@ public sealed partial class TokenCacheNotificationArgs
             PiiLoggingEnabled = piiLoggingEnabled;
         }
 
+        /// <summary>
+        /// This constructor is for test purposes only. It allows apps to unit test their MSAL token cache implementation code.
+        /// </summary>
+        public TokenCacheNotificationArgs(    // only use this constructor in product code
```

Comment:
We should not introduce a new ctor. Existing constructors are sufficient.

Reply:
This is a public class so I believe Neha's intention was to avoid breaking changes. I guess I can add it with a default value.